src/libraries/Microsoft.Extensions.Logging.Console/src/Microsoft.Extensions.Logging.Console.csproj

```diff
@@ -9,6 +9,7 @@
     <!-- Use targeting pack references instead of granular ones in the project file. -->
     <DisableImplicitAssemblyReferences>false</DisableImplicitAssemblyReferences>
     <IncludePlatformAttributes>true</IncludePlatformAttributes>
+    <EnableDllImportGenerator>true</EnableDllImportGenerator>
```

Comment:
Was this project being filtered out by for enabling in `generators.targets`? Is this something we should update in how the generator is enabled by default instead?

Reply:
This is a really wonky project. Adding InteropServices or the other two projects cause a lot of issues references. This was the only way I could make it work.

Reply:
I suspect it has to do with `DisableImplicitAssemblyReferences` two lines above.  I expect that means it already has the everything referenced. So it is probably actually more 'normal' than the other libraries projects :P

This seems to be the only project that has DisableImplicitAssemblyReferences=false and uses things from Common\Interop (a couple others do seem to have DllImports directly in their own source files), so I guess the one-off is fine.